pkg/certificates/bootstrap/manager.go

```diff
@@ -0,0 +1,19 @@
+package bootstrap
+
+import (
+	"crypto/x509"
+
+	"k8s.io/client-go/util/certificate"
+)
+
+const (
+	CertBytesValue        = "cert-bytes"
+	KeyBytesValue         = "key-bytes"
+	SigningCertBytesValue = "signing-cert-bytes"
```

Comment:
In my experience, these are often called `tls.crt`, `tls.key, and `ca.crt` respectively.  Like here:  https://github.com/kubernetes/api/blob/master/core/v1/types.go#L5584-L5586

Reply:
Done.